pkg/cmd/audit/audit_filter.go

```diff
@@ -231,6 +231,10 @@ func URIToParts(uri string) (string, schema.GroupVersionResource, string, string
 			uri = uri[1:]
 		}
 	}
+
+	// some request URL has query parameters like: /apis/image.openshift.io/v1/images?limit=500&resourceVersion=0
+	// we are not interested in the query parameters.
+	uri = strings.Split(uri, "?")[0]
```

Comment:
This works fine, but why not use standard `url.Parse`?

```
       u, err := url.Parse(uri)
	if err != nil {
		return ns, gvr, name, ""
	}
        fmt.Println(u.Path)
```
https://play.golang.org/p/KklTAOukAQB